src-control/Control/RefCount.hs

```diff
@@ -468,12 +472,24 @@ checkForgottenRefs = do
 #ifndef NO_IGNORE_ASSERTS
     return ()
 #else
-    performMajorGC
+    -- The hope is that by combining `performMajorGC` with `yield` that
+    -- the former starts the GC threads and the latter puts the current
```

Comment:
I suppose technically the former starts weak pointer finaliser threads, not GC threads.

Reply:
Fixed and clarified the comment in general.